### PR TITLE
remove flatpak_base_image

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

File: osbs/build/user_params.py [hunk 1]
```diff
@@ -126,14 +124,10 @@ def set_params(self,
         self.koji_parent_build.value = koji_parent_build
         self.koji_upload_dir.value = koji_upload_dir
         self.flatpak.value = flatpak
-        self.flatpak_base_image.value = flatpak_base_image
```
Comment (mlangsdorf) [inline osbs/build/user_params.py]: You'll need to change atomic_reactor/schemas/user_params.json in atomic-reactor

File: osbs/build/spec.py [hunk 2]
```diff
@@ -189,7 +189,6 @@ def __init__(self):
         self.koji_kerberos_principal = BuildParam("koji_kerberos_principal", allow_none=True)
         self.koji_kerberos_keytab = BuildParam("koji_kerberos_keytab", allow_none=True)
         self.flatpak = BuildParam("flatpak", default=False)
-        self.flatpak_base_image = BuildParam("flatpak_base_image", allow_none=True)
```
Comment (mlangsdorf) [inline osbs/build/spec.py]: I don't think you want to change build_request v1 here - that changes the behavior of Arrangement 5- systems.

Comment (twaugh) [reply osbs/build/spec.py]: I think we're fine with making changes that only affect flatpak builds, as they are not working prior to arrangement 6 in any case.

File: osbs/build/user_params.py (hunk 1 above)
Comment (rcerven) [reply osbs/build/user_params.py]: ok will make that change

**Comment by MartinBasti** [comment]
Release notes:

> - flatpak base image is now properly loaded from a reactor-config-map

**Pull request merged**